web/src/routes/(user)/albums/+page.svelte

```diff
@@ -86,16 +104,34 @@
 
   $: {
     const { sortBy } = $albumViewSettings;
-    if (sortBy === 'Most recent photo') {
+    if (sortBy === sortByOptions.mostRecent.dropDow) {
```

Comment:
This looks like a nightname lol. Is there any way to clean this up?

Reply:
We can hardcode the value like before, but it may be complicated if we add labels or other ways of sorting. :man_shrugging: 

If you have any other idea, I'm in!